### PR TITLE
Remove cherry-picking drag prompt

### PR DESCRIPTION
**Pull request by tidy-dev**
## Description

Remove cherry-picking drag prompt used when introducing drag and drop. We no longer think it is necessary as drag/drop discoverability doesn't appear to be a problem and now is not the only drag and drop operation.

### Screenshots
**REMOVED** tooltip/prompt shown in below image:
image
## Release notes

Notes: [Removed] Cherry-pick drag prompt removed.

### CODE REVIEW

**Review by billygriffin — DISMISSED** [review]
Sounds good! 🪓

File: app/src/ui/history/selected-commit.tsx [hunk 1]
```diff
@@ -276,7 +275,7 @@ export class SelectedCommit extends React.Component<
       return null
     }
 
-    return <DragOverlay dragZoneDescription="branch-button" />
+    return <div id="drag-overlay"></div>
```
Comment (sergiou87) [inline app/src/ui/history/selected-commit.tsx]: Why didn't you remove this function (and the `showDragOverlay` prop) altogether? 🤔 

Just keeping changes to a minimum?

**Review by sergiou87 — DISMISSED** [review]
LGTM, but I'm curious to know why not removing all related to the `renderDragOverlay` method 🤔

File: app/src/ui/history/selected-commit.tsx (hunk 1 above)
Comment (tidy-dev) [reply app/src/ui/history/selected-commit.tsx]: We still want to only show the drag overlay when this.props.showDragOverlay. I think I am misunderstanding you..

**Review by sergiou87 — APPROVED** [review]
:shipit:

**Pull request merged**